aiohttp/web_response.py

```diff
@@ -398,7 +402,7 @@ async def _do_start_compression(self, coding: ContentCoding) -> None:
         if coding != ContentCoding.identity:
             assert self._payload_writer is not None
             self._headers[hdrs.CONTENT_ENCODING] = coding.value
-            self._payload_writer.enable_compression(coding.value)
+            self._payload_writer.enable_compression(coding.value, self._strategy)
```

Comment:
Type of `self._payload_writer` is `AbstractStreamWriter`, which doesn't support this 2nd parameter:
https://github.com/aio-libs/aiohttp/blob/3.9/aiohttp/abc.py#L184

Reply:
So, my question is where is the parameter actually used?

Reply:
Regardless of whether it works in your test, nothing can be merged until mypy passes successfully, and currently this will cause a type error.

I'm not too familiar with this code, so I'm not sure if the type can be changed to `StreamWriter`, or this parameter added to `AbstractStreamWriter`, or if we would need an `isinstance(StreamWriter, ...)` check. Maybe @asvetlov can answer that one.